docs/source/user-guide/tutorials/build-pkgs.rst

```diff
@@ -99,7 +98,7 @@ Editing the meta.yaml file
        license:
        license_file:
```

Comment:
I think license_file can be removed

Reply:
Do I remove lines 99 and 100 or just the license_file: ?

Reply:
I think it's fine to remove both.  Those are not required fields, and simplifying the example is probably a good thing.